spec/lib/process_settings/file_monitor_spec.rb

```diff
@@ -5,7 +5,7 @@
 require 'support/shared_examples_for_monitors'
 
 describe ProcessSettings::FileMonitor do
-  SETTINGS_PATH = "./settings.yml"
+  SETTINGS_PATH = "./settings#{Thread.current.object_id}.yml"
```

Comment:
Does this code run multi-threaded now?

Reply:
In the jenkinsfile there are multiple versions of the tests all running at the same time, so I moved the majority of tests to being able to use `Tempfile`, but this test was proving difficult since it's using a constant.  So I just added the thread id to make sure the different appraisals don't step on each other.

Reply:
But they aren't running as multiple threads in the same process, are they? It sounds like they are multiple processes on the same VM? It seems like the Process Id is what you'd want in that case.

Reply:
fair point, switched